docs/installation-and-operations/bim-edition/README.md

```diff
@@ -17,27 +17,12 @@ An existing OpenProject on-premises (self hosted) installation can easily be swi
 
 Switching to the BIM Edition will not affect your existing data. Your team will be able to continue working just as before. By switching to the BIM edition additional features will become available  when you activate the "BCF" module in the [project's settings](../../user-guide/projects/project-settings/modules).
 
-
-
-## Docker-based installation
```

Comment:
CC @oliverguenther As far as I can tell this removal is intended to be only reverted in 16.6, but on dev we want to keep the docs.

Can you please confirm and probably revert the revert or cherry-pick the docs once more?